docs/content/v2024.2/yugabyte-platform/back-up-restore-universes/disaster-recovery/disaster-recovery-setup.md

```diff
@@ -177,7 +177,8 @@ For more information on alerting in YugabyteDB Anywhere, refer to [Alerts](../..
 
 ## Add a database to an existing DR
 
-Note that, although you don't need to create objects on the DR replica during initial DR setup, when you add a new database to an existing DR configuration, you _do_ need to create the same objects on the DR replica. If DR primary and replica objects don't match, you won't be able to add the database to DR.
+On the DR replica, create a database with the same name as that on the DR primary. In [Manual Schema Changes](../#manual-schema-changes) mode, you don't need to create objects on the DR replica. In this mode, DR performs a full copy of the data to be replicated on the DR primary, and automatically creates tables and objects, and restores data on the DR replica from the DR primary. However, in [Semi-automated Schema Changes](../#semi-automated-schema-changes) mode, you need to create all objects (tables, indexes etc) on the DR replica exactly the same way as the DR primary prior to setting up xcluster DR.
```

Comment:
@shahrooz1997 : @charleswang234 : Not sure why we had the previous comment here. We don't need to create objects on the DR primary in non-DB-scoped mode correct?

Reply:
Yes, we don't need to have the same database on the target side. I guess this text were referring to a case where there tables exist on the other side but their schema is different in which case both initial setup and add database would fail.